### PR TITLE
Add spanish translation

### PR DESCRIPTION
**Pull request by haggen88**
@vixalien

### CODE REVIEW

File: po/LINGUAS [hunk 1]
```diff
@@ -8,4 +9,4 @@ oc
 pt_BR
 ru
 ta
-tr
\ No newline at end of file
+tr
```
Comment (vixalien) [inline po/LINGUAS:12]: why did this line change?

File: po/es.po [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+msgid "Strikethrough"
+msgstr "Tachado"
+
+#: data/ui/window.ui:138
+msgid "All Notes"
+msgstr "Todas las notas"
+
+#. TRANSLATORS: eg. 'Translator Name <your.email@domain.com>' or 'Translator Name https://website.example'
+#: src/application.ts:371
+msgid "translator-credits"
```
Comment (vixalien) [inline po/es.po:280]: Write your names here

**Review by vixalien — CHANGES_REQUESTED** [review]

File: po/LINGUAS (hunk 1 above)
Comment (haggen88) [reply po/LINGUAS:12]: @vixalien I guess adding "es" (in alphabetical order) shifts "tr" one line.

Comment (vixalien) [reply po/LINGUAS:12]: no, it's about an added endline at the bottom of the file. no worries though

**Review by vixalien — APPROVED** [review]

**Pull request merged**